notebooks/commercial_building_recovery.ipynb

```diff
@@ -0,0 +1,169 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-09-29T15:02:55.785647Z",
+     "start_time": "2023-09-29T15:02:55.777915Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from pyincore import IncoreClient, RepairService, MappingSet\n",
+    "from pyincore.analyses.commercialbuildingrecovery.commercialbuildingrecovery import CommercialBuildingRecovery\n",
+    "\n",
+    "import pyincore.globals as pyglobals"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Connect to IN-CORE service\n",
+    "client = IncoreClient(pyglobals.INCORE_API_DEV_URL)"
```

Comment:
This notebook should point to incore-prod

Reply:
you might also need to migrate those ids to prod

Reply:
Sorry last thing, if you are releasing any new datasets, you will need to let @ywkim312 know